packager/certloader/certloader.go

```diff
@@ -41,7 +41,7 @@ func PopulateCertCache(config *util.Config, key crypto.PrivateKey, developmentMo
 			return nil, errors.Wrapf(err, "checking %s", config.CertFile)
 		}
 	}
-	certCache := certcache.New(certs, config.OCSPCache)
+	certCache := certcache.New(certs, config.OCSPCache, developmentMode)
```

Comment:
I think you need to modify the certcache instantiation in gateway_server as well.